deploy/kubeturbo-for-gke/README.md

```diff
@@ -0,0 +1,94 @@
+### Prerequisites
+* Turbonomic 6.0+
+* Running Google Kubernetes cluster 1.6+ (Verified on 1.6, 1.7, and 1.8)
+
+### <a name="configFile"></a>Step One: Create a secret for kubeturbo configuration file
+
+Create a file called `config`, with the following contents:
+
+> The `<TURBONOMIC_SERVER_URL>` is typically `https://<TURBO_SERVER_IP>:443`
+
+> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `6.0`
+
+```json
+{
+	"communicationConfig": {
+		"serverMeta": {
+            "version": "<TURBONOMIC_SERVER_VERSION>",
+		    "turboServer": "<TURBONOMIC_SERVER_URL>"
+		},
+		"restAPIConfig": {
+			"opsManagerUserName": "<TURBONOMIC_USERNAME>",
+			"opsManagerPassword": "<TURBONOMIC_PASSWORD>"
+		}
+	},
+	"targetConfig": {
```

Comment:
I still think this part is a bit confusing. The username and password is confusing to the user.

Reply:
Yea. it is. Should we add notes here?
Max is working on this [issue](https://github.com/turbonomic/kubeturbo/issues/101). We can update this documentation after the issue get fixed.